core/src/main/resources/lib/form/nested.jelly

```diff
@@ -27,10 +27,10 @@ THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-	<tr>
-    <td colspan="2"/>
-    <td colspan="2">
+	<div class='tr'>
+    <div colspan="2"/>
+    <div colspan="2">
 	    <d:invokeBody />
-	  </td>
-	</tr>
-</j:jelly>
\ No newline at end of file
+	  </div>
+	</div>
```

Comment:
I think we are missing a `</div>` tag here, there are 3 opening div tags and only 2 closing ones

Reply:
This is another case of you thinking you were looking at html5, when in fact it was xml that would eventually be transformed to produce html. One of the two `div colspan="2"` nodes had a `/>` and thus didn't need a `</div>`.

That said, if this was too confusing for you, it could easily be too confusing for someone else, so I'm not terribly concerned about switching away from `/>` notation, however, in principle if that's a thing, it would be better applied to the entire repository (or even all of jenkinsci by JEP or something) than by forcing me to deal w/ it while I try to do this work.

Reply:
I'm sorry if this is something that should not have been highlighted. I was asked to review this PR and that is why I gave these comments.

I am not a jelly expert but my understanding of it has always been that it does not touch tags that are not "jelly tags". Given that we use an html 5 doctype I jumped to the conclusion that we want to have html5 compliant markup. Browsers do a good job of "fixing" bad markup, but I believe that we should strive to have code that is correct so we don't misread it in the future (I for one thought that `<div />` was a typo for `</div>` when in fact it was just the equivalent of `<div></div>`.

Again, I think it would be best to have a third opinion, these are just my 2 cents.